### ISSUE TITLE
Uncaught TypeError: Cannot read property 'start' of undefined

### ISSUE THREAD

**Issue by phocke** [zhuochun/md-writer#52]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.0
**System**: Mac OS X 10.10.3
**Thrown From**: [markdown-writer](https://github.com/zhuochun/md-writer) package, v1.4.0

### Stack Trace

Uncaught TypeError: Cannot read property 'start' of undefined

```
At /Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/range.js:35

TypeError: Cannot read property 'start' of undefined
  at Function.module.exports.Range.fromObject (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/range.js:35:31)
  at TextBuffer.module.exports.TextBuffer.getTextInRange (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:373:36)
  at TextEditor.module.exports.TextEditor.getTextInRange (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:686:26)
  at InsertLinkView.module.exports.InsertLinkView.setFieldsFromSelection (/Users/phocke/.atom/packages/markdown-writer/lib/insert-link-view.coffee:80:25)
  at /Users/phocke/.atom/packages/markdown-writer/lib/insert-link-view.coffee:65:8
  at /Users/phocke/.atom/packages/markdown-writer/lib/insert-link-view.coffee:197:47
  at /Users/phocke/.atom/packages/markdown-writer/lib/insert-link-view.coffee:202:42
  at FSReqWrap.cb [as oncomplete] (fs.js:209:19)

```

### Commands

```
     -5:57.6.0 core:save (atom-text-editor.editor.is-focused.autocomplete-active)
  2x -5:56.3.0 core:move-down (atom-text-editor.editor.is-focused)
     -5:05.8.0 core:move-left (atom-text-editor.editor.is-focused)
  2x -5:05.4.0 core:move-down (atom-text-editor.editor.is-focused)
  5x -0:53.1.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -0:50.1.0 core:move-right (atom-text-editor.editor.is-focused)
  3x -0:48.3.0 core:move-left (atom-text-editor.editor.is-focused)
  3x -0:45.2.0 core:move-right (atom-text-editor.editor.is-focused.autocomplete-active)
 12x -0:44.5.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:38.2.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:35.1.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -0:33.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:33.8.0 markdown-writer:insert-link (atom-text-editor.editor)
     -0:27.6.0 core:paste (atom-text-editor.editor.mini.is-focused)
     -0:14.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:01.7.0 markdown-writer:insert-link (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "kobalt-ui",
      "kobalt"
    ],
    "ignoredNames": [
      ".git",
      ".svn",
      ".DS_Store",
      "*.log",
      "*.csv"
    ],
    "excludeVcsIgnoredPaths": false
  }
}
```

### Installed Packages

```coffee
# User
Sublime-Style-Column-Selection, v1.3.0
advanced-new-file, v0.4.3
copy-filename, v1.0.1
file-icons, v1.5.8
git-plus, v5.2.2
kobalt, v0.5.0
kobalt-ui, v0.4.0
language-log, v0.4.0
markdown-writer, v1.4.0
merge-conflicts, v1.3.2
minimap, v4.10.0
minimap-git-diff, v4.1.3
pigments, v0.8.1

# Dev
No dev packages
```

**Comment by zhuochun** [comment]
Would you add some steps to reproduce this bug? Thanks.

**Comment by zhuochun** [comment]
Likely it is solved at release `v1.6.0`.

**Comment by zhuochun** [comment]
Aah, encountered this myself. It might be a bug in Atom https://github.com/atom/atom/issues/7961
